sdk/logical/acme_billing.go

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package logical
+
+import "context"
+
+type ACMEBillingSystemView interface {
```

Comment:
Not sure if I like this or if we want to rename it now to just `BillingSystemView` or `ActivityEventSystemView` or similar.